packages/suite/src/components/suite/WordInputAdvanced/index.tsx

```diff
@@ -92,55 +98,75 @@ const WordInputAdvanced = (props: Props) => {
-                        <ButtonPin type="button" data-value="9" onClick={() => onSubmit('9')} />
-                    </Row>
-                    <Row>
-                        <ButtonPin type="button" data-value="4" onClick={() => onSubmit('4')} />
-                        <ButtonPin type="button" data-value="5" onClick={() => onSubmit('5')} />
-                        <ButtonPin type="button" data-value="6" onClick={() => onSubmit('6')} />
-                    </Row>
-                    <Row>
-                        <ButtonPin
-                            type="button"
-                            data-value="1"
-                            onClick={() => onSubmit('1')}
-                            data-test="@recovery/word-input-advanced/1"
-                        />
-                        <ButtonPin type="button" data-value="2" onClick={() => onSubmit('2')} />
-                        <ButtonPin type="button" data-value="3" onClick={() => onSubmit('3')} />
-                    </Row>
-                </>
-            )}
+            <DeviceMatrixExplanation
+                items={[
+                    {
+                        key: '1',
+                        title: <Translation id="TR_RECOVERY_MATRIX_DISPLAYED_ON_TREZOR" />,
+                        deviceImage: 1,
+                    },
+                    {
+                        key: '2',
+                        title: (
+                            <TrezorLink variant="underline" href={URLS.WIKI_ADVANCED_RECOVERY}>
+                                <Translation id="TR_LEARN_ADVANCED_RECOVERY" />
+                            </TrezorLink>
+                        ),
+                        icon: 'INFO_ACTIVE',
+                    },
+                ]}
+            />
+            <MatrixWrapper>
+                {count === 9 && (
+                    <>
+                        <Row>
+                            <ButtonPin type="button" data-value="7" onClick={() => onSubmit('7')} />
+                            <ButtonPin type="button" data-value="8" onClick={() => onSubmit('8')} />
+                            <ButtonPin type="button" data-value="9" onClick={() => onSubmit('9')} />
+                        </Row>
+                        <Row>
+                            <ButtonPin type="button" data-value="4" onClick={() => onSubmit('4')} />
+                            <ButtonPin type="button" data-value="5" onClick={() => onSubmit('5')} />
+                            <ButtonPin type="button" data-value="6" onClick={() => onSubmit('6')} />
+                        </Row>
+                        <Row>
+                            <ButtonPin
+                                type="button"
+                                data-value="1"
+                                onClick={() => onSubmit('1')}
+                                data-test="@recovery/word-input-advanced/1"
+                            />
```

Comment:
total offtopic: this is nice example of how linters can be dumb. It would be much better if this was a single line consistent with all the other ones even at the expense of exceeding line length limit. .)